### ISSUE TITLE
Time availability TV broadcast

### ISSUE THREAD

**Issue by WebworkrNet** [mediathekview/zapp#362]
The media libraries usually indicate how long a programme is available in the media library. It would be desirable if this information were also captured by the upstream crawler. In addition to displaying this information, this could be used to remind the user. The ZDF Mediathek pushes a reminder several times within the last 24 hours before expiry, even after the device has been restarted.

**Comment by cemrich** [comment]
I don't think that the data needed is exposed by mediathekviewweb. I'll have to look into it.
